adapters/kagome/cmake/HunterConfig.cmake

```diff
@@ -18,7 +18,7 @@
 # The version of kagome to be tested
 hunter_config(kagome
   GIT_SUBMODULE hosts/kagome
-  CMAKE_ARGS TESTING=OFF
+  CMAKE_ARGS HUNTER_CONFIGURATION_TYPES=Release TESTING=OFF
```

Comment:
Is this the right way to go? I would much rather add this to the adapters `CMakeLists.txt` (or just `Makefile`). For example adding a `set(HUNTER_CONFIGURATION_TYPES ${CMAKE_BUILD_TYPE})` at the beginning of the main file should be enough.